main/src/ca/uwaterloo/flix/language/ast/NamedAst.scala

```diff
@@ -286,7 +286,7 @@ object NamedAst {
 
     case class Unit(loc: SourceLocation) extends NamedAst.Type
 
-    case class Enum(name: Symbol.EnumSym) extends NamedAst.Type
+    case class Enum(name: Symbol.EnumSym, arity: Int) extends NamedAst.Type
```

Comment:
Is there a reason to put the arity here instead of simply the entire kind?

Reply:
Not as far as I can tell.